### PR TITLE
Completion en_us + zh_cn

### PR DESCRIPTION
**Pull request by latiao-1337**

### CODE REVIEW

File: src/main/resources/assets/liquidbounce/lang/en_us.json [hunk 1]
```diff
@@ -394,6 +394,7 @@
   "liquidbounce.module.blink.description": "Makes it look as if you were teleporting to other players.",
   "liquidbounce.module.blockBounce.description": "Allows you to bounce higher on bouncy blocks.",
   "liquidbounce.module.blockESP.description": "Allows you to see selected blocks through walls.",
+  "liquidbounce.module.blockOutline.description": "Make the block have an outline.",
```
Comment (DataM0del) [inline src/main/resources/assets/liquidbounce/lang/en_us.json]:
```suggestion
  "liquidbounce.module.blockOutline.description": "Adds an outline to all blocks.",
```

File: src/main/resources/assets/liquidbounce/lang/en_us.json [hunk 2]
```diff
@@ -527,6 +528,7 @@
   "liquidbounce.module.plugins.messages.timeOut": "Plugins check timed out.",
   "liquidbounce.module.portalMenu.description": "Allows you to open menus while standing inside a portal.",
   "liquidbounce.module.projectilePuncher.description": "Shoots back incoming projectiles around you.",
+  "liquidbounce.module.ProphuntESP.description": "Make you see Prophunt .",
```
Comment (1zun4) [inline src/main/resources/assets/liquidbounce/lang/en_us.json]: Is it ProphuntESP as keyword? I think lowerCamelCase translates it to prophuntEsp or similiar.

Comment (EclipsesDev) [reply src/main/resources/assets/liquidbounce/lang/en_us.json]: "make you see prophunt" bruh hell naw, what tf is this grammar xd

**Comment by DataM0del** [comment]
lol

**Pull request closed without merging**